### DISCUSSION TITLE
Installation of maint-3.8 on Raspberry Pi4

### DISCUSSION THREAD

**Discussion by gerd0355** [duggabe/gr-control#12] [General]
I want to install  gr-control in version maint-3.8 on  Raspberry Pi 4B using OS Bullseye 64-Bit Legacy. When I installed the library libiio the command cmake .. generated an error:
CMake Error at CMakeLists.txt:467 (message):
  Unable to find libzstd dependency
Can you help me, please.

**Comment by duggabe** [comment]
I'm sorry, but you will have to update your computing platform to Version 3.10. There are no current plans to "backport" new code to the "maint" branches. Once you update, you get GNU Radio 3.10 with gr-iio including all dependencies from Debian.

**Comment by gerd0355** [comment]
Thank you for your answer. I have tried with Raspberry Pi 5 and OS
Bookworm. It works very good.

duggabe ***@***.***> schrieb am So., 23. Juni 2024, 23:42:
